homepage/views.py

```diff
@@ -53,3 +55,23 @@ def add_review(request):
     else:
         form = ReviewForm()
     return render(request, 'homepage/add_review.html', {'form': form})
+
+
+def sign_up(request):
+    if request.method == "POST":
+        form = SignUpForm(request.POST)
+
+        if form.is_valid():
+            appUser = AppUser()
+            appUser.user = form.save()
+            appUser.save()
+
+            login(request, appUser.user)
+            messages.success(request, "Registration successful.")
```

Comment:
Where is that message shown? if the user is redirected to the **sign_in** page

Reply:
Error messages are only partly implemented and will be fully implemented on a separate PR with other features
I've added a comment in the PR description for more clarity.